go.mod

```diff
@@ -70,5 +66,5 @@ require (
 	github.com/shirou/gopsutil/v3 v3.22.9
 	github.com/trinodb/trino-go-client v0.300.0
 	google.golang.org/genproto v0.0.0-20211129164237-f09f9a12af12 // indirect
-	gopkg.in/20210107192922/yaml.v3 v3.0.0-00010101000000-000000000000
+	gopkg.in/yaml.v3 v3.0.1
```

Comment:
is there a reason we specify only this version to be .1

Reply:
v3.0.1 is the latest stable version available. The previous thing was a workaround to pin it to one specific commit within v3.0.0, which is generally not something we'd want to do since individual commits aren't considered stable enough for release until they have a git tag release.

Reply:
`v3.0.1` is just stating the particular minimum version of `gopkg.in/yaml.v3` that's required. The version after the module name in the `go.mod` tells Go which version of the YAML module to use when `import gopkg.in/yaml.v3` is used in a source file.

Reply:
ahh okay
thank you! i hope i wasnt holding this up from being merged